contributor-docs/maintainers-guide.md

```diff
@@ -102,6 +102,28 @@ The remaining files can then be generated by running `sh python/generate_all_dep
 
 **Automated Updates**: The `.github/workflows/update-python-deps.yml` workflow automatically updates these Python dependencies weekly. You can also manually trigger this workflow to update all Python dependencies if needed. For an example of an automated dependency update PR, see https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/2676. 
 
+### Prerequisites for Generating Dependencies
```

Comment:
Thanks! Had one more thought - could we mention that https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/.github/workflows/update-python-deps.yml runs weekly now? @liferoad added it last week.

Otherwise, this LGTM - thank you!